core/config/router_struct.go

```diff
@@ -21,10 +21,11 @@ type RouteTag struct {
 
 // Match is checking source, source tags, and http headers
 type Match struct {
-	Refer       string                       `yaml:"refer"`
-	Source      string                       `yaml:"source"`
-	SourceTags  map[string]string            `yaml:"sourceTags"`
-	HTTPHeaders map[string]map[string]string `yaml:"httpHeaders"`
+	Refer          string                       `yaml:"refer"`
+	Source         string                       `yaml:"source"`
+	SourceTags     map[string]string            `yaml:"sourceTags"`
+	HTTPHeaders    map[string]map[string]string `yaml:"httpHeaders"`
```

Comment:
Consider about mutiple-protocol, we can not maintain config like this, and considering about perf ,we shouldn't use 3 level map , so unify all protocol to headers, and compatible with httpHeaders config for future release